watertap/unit_models/nanofiltration_DSPMDE_0D.py

```diff
@@ -1859,29 +2009,26 @@ def calculate_scaling_factors(self):
                 iscale.set_scaling_factor(
                     self.feed_side.mass_transfer_term[0, "Liq", j], 1
                 )
```

Comment:
Since there are constraints to calculate the mass transfer, would it be better to approximate a scaling factor based on some formulaic representation of the expected values for mass transfer terms? I imagine that mass transfer rates could be very different for different ions